perfkitbenchmarker/linux_benchmarks/netperf_aggregate_benchmark.py

```diff
@@ -160,24 +192,36 @@ def RunNetperfAggregate(vm, server_ips):
       login_shell=False,
       timeout=1200)
 
-  # print out netperf_tps.log to log
-  stdout_1, stderr_1 = vm.RemoteCommand(
-      f'cat {netperf.NETPERF_EXAMPLE_DIR}/netperf_tps.log',
-      ignore_failure=True,
-      should_log=True,
-      login_shell=False,
-      timeout=1200)
-
-  logging.info(stdout_1)
-  logging.info(stderr_1)
-
+  samples = []
   # do post processing step
-  proc_stdout, _ = vm.RemoteCommand(
-      f'cd {netperf.NETPERF_EXAMPLE_DIR} && ./post_proc.py '
+  if 'STREAM' in FLAGS.netperf_aggregate_benchmarks:
+    proc_stdout, _ = vm.RemoteCommand(
+      f'cd {netperf.NETPERF_EXAMPLE_DIR} && python3 post_proc.py '
+      '--intervals netperf_outbound.log',
+      ignore_failure=False)
+    samples.extend(ParseNetperfAggregateOutput(proc_stdout, 'Outbound'))
+    vm.RemoteCommand(f'cd {netperf.NETPERF_EXAMPLE_DIR} && rm netperf_outbound*')
+  if 'MAERTS' in FLAGS.netperf_aggregate_benchmarks:
+    proc_stdout, _ = vm.RemoteCommand(
+      f'cd {netperf.NETPERF_EXAMPLE_DIR} && python3 post_proc.py '
+      '--intervals netperf_inbound.log',
+      ignore_failure=False)
+    samples.extend(ParseNetperfAggregateOutput(proc_stdout, 'Inbound'))
+    vm.RemoteCommand(f'cd {netperf.NETPERF_EXAMPLE_DIR} && rm netperf_inbound*')
+  if 'RRAGG' in FLAGS.netperf_aggregate_benchmarks:
+    proc_stdout, _ = vm.RemoteCommand(
+      f'cd {netperf.NETPERF_EXAMPLE_DIR} && python3 post_proc.py '
       '--intervals netperf_tps.log',
-      ignore_failure=True)
-
-  samples = ParseNetperfAggregateOutput(proc_stdout)
+      ignore_failure=False)
+    samples.extend(ParseNetperfAggregateOutput(proc_stdout, 'Request/Response Aggregate'))
+    vm.RemoteCommand(f'cd {netperf.NETPERF_EXAMPLE_DIR} && rm netperf_tps*')
+  if 'BIDIR' in FLAGS.netperf_aggregate_benchmarks:
```

Comment:
I think you can condense the code here to be more succinct:

BENCHMARK_INTERVAL_MAPPING = {
  'BIDIR': IntervalNaming('netperf_bidirectional', 'netperf_bidirectional*'),
}
...
for benchmark in FLAGS.netperf_aggregate_benchmarks:
  intervals_flag = BENCHMARK_INTERVAL_MAPPING[benchmark].intervals_flag
  output_file = BENCHMARK_INTERVAL_MAPPING[benchmark].output_file
  proc_stdout, _ = f'cd {netperf.NETPERF_EXAMPLE_DIR} && python3 post_proc.py '
                               f'--intervals {intervals_flag}.log', ignore_failure=False)
  samples.extend(ParseNetperfAggregateOutput(proc_stdout, f'{output_file}'))
  
Something like this but with better names.